commands/create_kubeconfig.go

```diff
@@ -452,43 +407,28 @@ func createKubeconfig(args createKubeconfigArguments) (createKubeconfigResult, e
 		}
 	} else {
 		// create a self-contained kubeconfig
-		kubeconfig := Kubeconfig{
-			APIVersion:     "v1",
-			Kind:           "Config",
-			CurrentContext: args.contextName,
-			Clusters: []KubeconfigNamedCluster{
-				{
-					Name: "giantswarm-" + args.clusterID,
-					Cluster: KubeconfigCluster{
-						Server:                   result.apiEndpoint,
-						CertificateAuthorityData: base64.StdEncoding.EncodeToString([]byte(response.Payload.CertificateAuthorityData)),
-					},
-				},
-			},
-			Contexts: []KubeconfigNamedContext{
-				{
-					Name: args.contextName,
-					Context: KubeconfigContext{
-						Cluster: "giantswarm-" + args.clusterID,
-						User:    "giantswarm-" + args.clusterID + "-user",
-					},
-				},
-			},
-			Users: []KubeconfigUser{
-				{
-					Name: "giantswarm-" + args.clusterID + "-user",
-					User: KubeconfigUserKeyPair{
-						ClientCertificateData: base64.StdEncoding.EncodeToString([]byte(response.Payload.ClientCertificateData)),
-						ClientKeyData:         base64.StdEncoding.EncodeToString([]byte(response.Payload.ClientKeyData)),
-					},
+		var yamlBytes []byte
+		{
+			c := k8srestconfig.Config{
+				Logger: microloggertest.New(),
```

Comment:
We should only use the `microloggertest` package in unit tests. Lets just add a regular micrologger.